### PR TITLE
Vulkan

### PR DESCRIPTION
**Pull request by JPeterMugaas**

### CODE REVIEW

File: mingw-w64-vulkan [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Maintainer: Some One <some.one@some.email.com>
```
Comment (mati865) [inline mingw-w64-vulkan]: I think you missed it.

File: mingw-w64-vulkan [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+_realname=vulkan
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}-headers" "${MINGW_PACKAGE_PREFIX}-${_realname}-man-pages")
+_pkgname=Vulkan-Docs
+_pkgver=1.0-core+wsi-20160527
+epoch=1
+pkgrel=1
+arch=('any')
+url="https://www.khronos.org/vulkan/"
+makedepends=(asciidoc)
```
Comment (mati865) [inline mingw-w64-vulkan]: Depends on `${MINGW_PACKAGE_PREFIX}-libxslt` also.

**Comment by maoueh** [comment]
This will probably close #1168.

**Comment by JPeterMugaas** [comment]
Perhaps but I still think you might need the library.  There is something at:
https://vulkan.lunarg.com/signin

But I have not yet done a package for it.  I think you just need the .DLL and that is part of the driver or the development SDK.  I do know that when I built glfew, cmake did detect the vulkan .DLL and compiled a test for it.

**Comment by mati865** [comment]
@maoueh 
They are just headers, we would need Vulkan validation layers also.
But their code depend on MSVC for Windows so it would require a lot patches to compile it inside MSYS.

**Pull request merged**